LocalPayment/src/main/java/com/braintreepayments/api/LocalPaymentClient.java

```diff
@@ -32,7 +33,7 @@ public class LocalPaymentClient {
     /**
      * Create a new instance of {@link LocalPaymentClient} from within an Activity using a {@link BraintreeClient}.
      *
-     * @param activity a {@link FragmentActivity}
+     * @param activity        a {@link FragmentActivity}
```

Comment:
The spacing for this looks different then the `braintreeClient` right below. Is this intentional?

Reply:
Yeah it's the auto formatter in Android studio. We would need to run it on each file for 100% consistency.